android-runtime/.idea/.name

```diff
@@ -0,0 +1 @@
+Sdkgen Kotlin Android Runtime
```

Comment:
This file can be removed

Reply:
It's used by android studio to show a soft name of the project in some view options of the files browser. don't believe it needs to be removed 🤔